esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2921,80 +2832,83 @@ bool CFileSprayEx::onFileList(IEspContext &context, IEspFileListRequest &req, IE
         if (!path || !*path)
             throw MakeStringException(ECLWATCH_INVALID_INPUT, "Path not specified.");
 
-        double version = context.getClientVersion();
-        const char* netaddr = req.getNetaddr();
-        if (!netaddr || !*netaddr)
-            throw MakeStringException(ECLWATCH_INVALID_INPUT, "Network address not specified.");
-        const char* fileNameMask = req.getMask();
-        bool directoryOnly = req.getDirectoryOnly();
-        PROGLOG("FileList:  Netaddr %s, Path %s", netaddr, path);
-
         StringBuffer sPath(path);
         const char* osStr = req.getOS();
-        if (osStr && *osStr)
+        if (!isEmptyString(osStr))
         {
             int os = atoi(osStr);
             const char pathSep = (os == OS_WINDOWS) ? '\\' : '/';
             sPath.replace(pathSep=='\\'?'/':'\\', pathSep);
         }
         addPathSepChar(sPath);
 
+        //The FileSpray.FileList is mainly used to list dropzone files.
+        //In a bare metal environment, the FileSpray.FileList is also used to retrieve a list of log files for backward compatible.
```

Comment:
>FileSpray.FileList is also used to retrieve a list of log files for backward compatible.

is there another route in bare-metal?
If not comment should probably just say "FileSpray.FileList is also used to retrieve a list of log files" for now.

Reply:
There is no another route in bare-metal for now. Will change.